src/app-layout/content-wrapper/styles.scss

```diff
@@ -14,3 +15,13 @@
     padding-right: awsui.$space-l;
   }
 }
+
+.content-type-dashboard {
+  margin-left: auto;
+  margin-right: auto;
+  @each $breakpoint, $width in constants.$adaptive-content-widths {
```

Comment:
What is the advantage of using these three breakpoints versus just having one max-width and letting the component width flexibly adapt when the viewport is narrower than that?

Reply:
It was design decision. I can tell more details in DM